tests/Fixer/LanguageConstruct/DeclareEqualNormalizeFixerTest.php

```diff
@@ -87,6 +87,11 @@ public function provideFixCases(): array
                 null,
                 ['space' => 'none'],
             ],
+            'declare having multiple directives' => [
```

Comment:
let's have this case for both, `space=none` and `space=single`

Reply:
done